config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
We could create `qemu-q35-system-x86_64` as a binary (which has symmetry with the qemu-lite variant), or maybe we could make that a symlink to something like `/usr/clear-containers/bin/qemu-system-x86_64`? That latter option has strong precedent too in a sense as we already have:

```
/etc/clear-containers/
/usr/share/clear-containers/
/usr/libexec/clear-containers/
/var/lib/clear-containers/
```